deploy/cr.yaml

```diff
@@ -37,6 +37,9 @@ spec:
     size: 3
     image: percona/percona-xtradb-cluster:8.0.23-14.1
     autoRecovery: true
+#    hookScript: |
```

Comment:
@spron-in Should  we add it to CR as the example? It is feature for our support team which will be used for debug. And what about option name. Is it ok? Or we need to add something like:
```
hookScripts:
  hook1: |
  hook2: |
```
  because as you remember support team wanted to have several scripts to entrypoints but we agreed that we will start from one :)

Reply:
No need to add it to default CR. Reasoning: It is internal functionality, no need to make our CR more scary :)

We are going to have only one script only, no plans to add more.

Reply:
@pooknull please remove it from CR